Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/type/rapidantigen/RATestProcessor.kt

```diff
@@ -276,3 +261,19 @@ private fun CoronaTestResult.toValidatedResult(): CoronaTestResult {
         RAT_INVALID
     }
 }
+
+// After 60 days, the previously EXPIRED test is deleted from the server, and it may return pending again.
+fun check60DaysRAT(test: BaseCoronaTest, newResult: CoronaTestResult, now: Instant): CoronaTestResult {
```

Comment:
Create only 1 method for both types?

Reply:
Both methods are different from each other, even if the name looks the same